### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by thedarkdivinity**
made a landing page.Images are random as of now but when i'll get images from the client,I'll update

### CODE REVIEW

File: client/src/components/landingpage/AniCard.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    height: 300,
+    width:"100%"
+  },
+  adjust: {
+      marginLeft:"auto",
+      marginRight:"auto"
+  }
+});
+
+export default function AniCard({number}) {
```
Comment (ShubhankarKG) [inline client/src/components/landingpage/AniCard.js:25]: Is the `number` prop required? It didn't have any use in the component, so we can safely remove it.

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]
Thanks! The UI is fantastic, apart from the inline changes suggested, it would be much better if we split up App.css into multiple css files, so that the css used for About is in About.css, and so on.

File: client/.eslintcache [hunk 2]
```diff
@@ -0,0 +1 @@
.../Desktop/CA-Website/client/src/components/landingpage/AniCard.js",[],"/home/sayush/Desktop/CA-Website/client/src/components/landingpage/CardCarousel.js",[],"/home/sayush/Desktop/CA-Website/client/src/components/landingpage/Landing.js",[],"/home/sayush/Desktop/CA-Website/client/src/components/Navbar.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/index.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/reportWebVitals.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/App.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/components/Navbar.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/components/landingpage/Landing.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/components/landingpage/CardCarousel.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/components/landingpage/Hero.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/components/landingpage/About.js",[],"/home/sayush/Desktop/chicken/CA-Website/client/src/components/landingpage/AniCard.js",[],{"ruleId":"78","replacedBy":"79"},{"ruleId":"80","replacedBy":"81"},"no-native-reassign",["82"],"no-negated-in-lhs",["83"],"no-global-assign","no-unsafe-negation"]
```
[the remark this answers is not in the document]
Comment (thedarkdivinity) [reply client/.eslintcache:1]: yes shubhankar understood

**Comment by BhavyaSheth22** [comment]
Closing as changes included in pr #29

**Pull request closed without merging**